front/app/components/LandingPages/citizen/HeaderContent.ts

```diff
@@ -88,6 +85,12 @@ export const HeaderSubtitle = styled.h2<{
   margin: 0;
   margin-bottom: 15px;
 
+  ${media.phone`
```

Comment:
After:
![image](https://user-images.githubusercontent.com/91883434/207950464-9fd216b3-b20f-4233-863a-1da1d412622e.png)

Reply:
We need more margin-bottom on the sub-header text. The button is too close to it.

It should be similar to when we have the avatar bubbles: if the avatar bubbles are the last element before the button, they have more margin-bottom. Add a similar margin bottom to the subheader text if there are no avatar bubbles.

Reply:
I suppose this issue wasn't introduced in this PR, was it? In this case, could we do it in a separate PR?

We already have logic for it, but it works only for the  `displayHeaderAvatar` setting https://github.com/CitizenLabDotCo/citizenlab/blob/b816374a55d9dfd3e6be81d0ea6e42cd024a116e/front/app/components/LandingPages/citizen/HeaderContent.ts#L91-L94
I'm not sure how to make it work with the conditions inside of the `AvatarBubbles` component.